packages/.eslintrc.json

```diff
@@ -14,7 +14,7 @@
             "error",
             { "paths": [{ "name": "lodash", "message": "Please use lodash-es instead." }] }
         ],
-        "react-hooks/exhaustive-deps": "warn",
+        "react-hooks/exhaustive-deps": "off",
```

Comment:
so why not just remove that rule?

Reply:
current, we only can ignore all warning message on CI